### ISSUE TITLE
Component error. Cannot setup on 2021.12

### ISSUE THREAD

**Issue by dronix27** [Limych/ha-narodmon#53]
**Environment**

- Home Assistant Core release with the issue: 2021.12.1
- This custom component release with the issue: 1.1.1
- Last working this custom component release (if known): 1.1.1
- Operating environment (Home Assistant/Supervisor/Docker/venv): Home Assistant Supervised

**Describe the bug**

The integration cannot setup on HA startup. There is an error in the log:
Error occurred loading configuration flow for integration narodmon: cannot import name 'HTTP_OK' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)

**Configuration.yaml**
```yaml

Add your configs here if any.
narodmon:
  apikey: !secret narodmon_apikey
  devices:
    - name: "Narodmon"
```

**Steps to Reproduce**

Upgrade HA to 2021.12

**Expected behavior**

The integration is loaded on HA startup.

**Debug log**

```text

Add your logs here.

```

**Additional context**

**Comment by yarodin** [comment]
Same story for Home Assistant 2021.12.3
"Setup failed for narodmon: Unable to import component: cannot import name 'HTTP_OK' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)"

**Comment by alexsheyko** [comment]
кому срочно, изменить

custom_components/narodmon/api.py

```
#from homeassistant.const import ATTR_ID, HTTP_OK
from homeassistant.const import ATTR_ID


                    #if resp.status != HTTP_OK:
                    if resp.status != 200:
```

**Comment by Limych** [comment]
Fixed in #46
